### PR TITLE
Flutter

### PR DESCRIPTION
**Pull request by kenvandine**
## Done

Adds Flutter to the first-run

There is a FIXME in place of the flutter logo, which seems to be causing the test failures.  I'd recommend using this logo:  https://drive.google.com/file/d/1Ka-_VgfVS_eLeY6DMZMqsFxFgr_fnvFf/view?usp=sharing

## Issue / Card

Fixes #

## QA

- Pull the branch
- Run the site using the dotrun snap with `$ dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [List additional steps to QA the new features or prove the bug has been resolved]

## Screenshots

[if relevant, include a screenshot]

### CODE REVIEW

**Comment by lilyvidenova** [comment]
@tbille - the demo is showing me an internal server error :( could you take a look, please?

**Comment by lilyvidenova** [comment]
@kenvandine - could I ask you to please create a similar piece of content about flutter for the beginning of the flow:
image
 


Unrelated to this, may I suggest we rearrange the languages, looking at the heatmap below
image

the most popular choices are Python, C, and Java - can we place them to be the first three as follows:

Python      C/C++     Pre-built apps ///and so on      

Java        Flutter

**Comment by kenvandine** [comment]
> @kenvandine - could I ask you to please create a similar piece of content about flutter for the beginning of the flow:
> image

I had done this but missed an include, I just pushed a fix for that.

> 
> Unrelated to this, may I suggest we rearrange the languages, looking at the heatmap below
> image
> 
> the most popular choices are Python, C, and Java - can we place them to be the first three as follows:
> 
> Python C/C++ Pre-built apps ///and so on
> 
> Java Flutter

Re-ordering should probably be done separately.

**Comment by lilyvidenova** [comment]
+1 🚀

File: templates/home/_fsf_flutter.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+<b>parts</b>:
+  <b>flutter-gallery</b>:
+    <b>plugin</b>: flutter
+    <b>source</b>: https://github.com/flutter/gallery.git
+    <b>flutter-target</b>: lib/main.dart
+
+ <b>apps</b>:
+   <b>flutter-gallery</b>:
+      <b>command</b>: gallery
+      <b>extensions</b>: [flutter-master]
```
Comment (solazio) [inline templates/home/_fsf_flutter.html:40]:
```suggestion
      <b>extensions</b>: [flutter-master]</pre>
```
Missing a `</pre>` here

File: templates/home/_fsf_flutter.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  </div>
+
+  <div class="col-6">
+    <h4>Here's how <a href="https://snapcraft.io/flutter-gallery">flutter-gallery</a> defines snapcraft.yaml:</h4>
+    <div class ="p-show-more is-collapsed" data-js="js-show-more">
+      <pre class="p-code-yaml"><b>name</b>: flutter-gallery
+<b>version</b>: "1.0"
+<b>summary</b>: Flutter Gallery
+<b>description</b>: |
+    A demo app for Flutter's material design and cupertino widgets, as well
```
Comment (solazio) [inline templates/home/_fsf_flutter.html:25]:
```suggestion
    A demo app for Flutter's material design [&hellip;]
```

As is just a description we keep it very brief.

**Pull request merged**